src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -311,11 +319,49 @@ public static void writeFileLines(Path file, List<String> lines) throws IOExcept
     public static void replaceInFile(Path file, String original, String replacement) throws IOException {
         InputStream inputStream = Files.newInputStream(file);
         List<String> lines = readFileLines(inputStream);
-        for (int i=0; i<lines.size(); i++)
+        for (int i = 0; i < lines.size(); i++) {
             lines.set(i, lines.get(i).replaceAll(original, replacement));
+        }
         writeFileLines(file, lines);
     }
 
+    /**
+     * Parses an xml file, looking for a given item within a given tag, and retrieves
+     * its value, if found
+     *
+     * @param fileName the full path of an xml file
+     * @param tag the tag name
+     * @param itemName the item name
+     * @return the value if found, null otherwise
+     * @throws IOException
+     */
+    public static String getNodeValue(String fileName, String tag, String itemName) throws IOException {
+        try {
+            File xmlFile = new File(Objects.requireNonNull(fileName));
+            if (!xmlFile.exists() || !fileName.endsWith(".xml")) {
+                throw new IOException("Not a valid file: " + fileName);
+            }
+
+            DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+            Document document = builder.parse(xmlFile);
+
+            NodeList nodeList = document.getElementsByTagName(tag);
+            for (int n = 0; n < nodeList.getLength(); n++) {
+                NamedNodeMap attributes = nodeList.item(n).getAttributes();
+                for (int i = 0; i < attributes.getLength(); i++) {
+                    Node item = attributes.item(i);
+                    String name = item.getNodeName();
+                    if (name != null && name.contains(itemName)) {
```

Comment:
Did you really mean `contains` instead of `equals`? If so, you might want to add this information in the javadoc.

Reply:
Yes, I meant `contains` (the reason is we can have different namespaces `android:...` or `a:...`)